### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by newtec-foggia** [oneduo/nova-file-manager#9] [Q&A] [answered]
Hi, I tried to install following the instructions, but inserting the field in the resource reports me
![Cattura](https://user-images.githubusercontent.com/83398776/183747084-b8d74d80-b063-4789-9dbe-c48ae1cdd14a.JPG)

**Comment by mikaelpopowicz** [comment]
Hi @newtec-foggia 

May you add some code snippet ? Maybe an excerpt of the stacktrace too ?

**Reply by newtec-foggia** [reply]
Hi @mikaelpopowicz 

![ERRORE](https://user-images.githubusercontent.com/83398776/184409435-cd2bc8dc-11e5-44fd-8098-9121bda80cec.JPG)

![Cattura1](https://user-images.githubusercontent.com/83398776/184410933-b0cd8937-870d-41d2-996e-eb9e9d4eb4b6.JPG)

![Cattura2](https://user-images.githubusercontent.com/83398776/184410953-828dfb24-6512-47ac-b83c-d6d10a69db10.JPG)

**Comment by mikaelpopowicz** [comment, marked as answer]
Hi @newtec-foggia 

I guess you are using Laravel 8.x, you may upgrade to latest release of this package which includes a `str` helper function.

**Reply by newtec-foggia** [reply]
upgraded to 0.2.2.. perfect thanks
